cmd/wallet/common.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import "github.com/kaspanet/kaspad/app/appmessage"
+
+const minConfirmations = 100
+
+func isUTXOSpendable(entry *appmessage.UTXOsByAddressesEntry, virtualSelectedParentBlueScore uint64) bool {
+	blockBlueScore := entry.UTXOEntry.BlockBlueScore
+	return blockBlueScore+minConfirmations < virtualSelectedParentBlueScore
```

Comment:
Do this only for coinbase outputs. All other outputs are spendable with 0 confirmations.
Also we have locktimes etc. but I guess it's too much to ask of you rn.